src/Autosuggest.js

```diff
@@ -215,6 +249,8 @@ export default class Autosuggest extends Component {
       focusedSectionIndex: sectionIndex,
       focusedSuggestionIndex: suggestionIndex
     });
+
+    this.onSuggestionFocused(sectionIndex, suggestionIndex);
```

Comment:
Hmm.. I'm sure I saw this being called in `onSuggestionMouseLeave()`. Must be a glitch in GitHub or my head :) Please ignore.